### PR TITLE
Test expanding all CF's 

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Some small column families are write stalling

#### Summary of Changes
Expand column families

Fixes #

### CODE REVIEW

**Comment by sagar-solana** [comment]
So this should just make the spikes taller and occur more spread out no?

**Comment by sakridge** [comment]
@carllin This seems to help a lot actually

**Comment by sagar-solana** [comment]
I guess now is as good a time as any to tune blocktree again. 

This is the PR that introduced those limits. 
https://github.com/solana-labs/solana/pull/5266

File: ledger/src/blocktree_db.rs [hunk 1]
```diff
@@ -679,27 +674,13 @@ impl<'a> WriteBatch<'a> {
...
-        _ => {
-            // We want smaller CFs to flush faster. This results in more WAL files but lowers
-            // overall WAL space utilization and increases flush frequency
-            options.set_write_buffer_size(MIN_WRITE_BUFFER_SIZE as usize);
-            options.set_target_file_size_base(MIN_WRITE_BUFFER_SIZE);
-            options.set_max_bytes_for_level_base(MIN_WRITE_BUFFER_SIZE);
-            options.set_level_zero_file_num_compaction_trigger(1);
-        }
-    }
+    // 512MB * 8 = 4GB. 2 of these columns should take no more than 8GB of RAM
```
Comment (sagar-solana) [inline ledger/src/blocktree_db.rs]: Fix this comment? It's 256MB and now we've got what 6 columns ?

**Pull request merged**